packages/core/src/types/config/fields.ts

```diff
@@ -20,7 +20,7 @@ export type FilterOrderArgs<ListTypeInfo extends BaseListTypeInfo> = {
 export type CommonFieldConfig<ListTypeInfo extends BaseListTypeInfo> = {
   access?: FieldAccessControl<ListTypeInfo>;
   hooks?: FieldHooks<ListTypeInfo, ListTypeInfo['fields']>;
-  label?: string;
+  label?: string; // TODO: move to ui?
```

Comment:
Apparently `text({ label: 'a text field' })` is what we use for fields, but not for lists.
Strange